.nancy-ignore

```diff
@@ -0,0 +1,21 @@
+# -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+# Dependencies used by github.com/gopherjs/gopherjs@v0.0.0-20210803090616-8f023c250c89
```

Comment:
Do we have bug stories in the backlog to fix these in the future? If not then let me know and we can create them

Reply:
not to my knowledge - happy to be shown how